docs/src/examples/components/Button/commonScreenerSteps.ts

```diff
@@ -0,0 +1,11 @@
+import { Button } from '@stardust-ui/react'
+
+const button = `.${Button.className}`
+
+const getScreenerSteps = (): ScreenerSteps => [
+  builder => builder.hover(button).snapshot('Hover state of the first button'),
```

Comment:
we usually use 3rd person and verbs for the snapshot text; not sure it's the best convention but let's keep consistent for now:
`.snapshot('Hovers the first button')`

same for all the other instances, I won't spam with comments 😄